nemo/collections/llm/gpt/model/base.py

```diff
@@ -310,6 +311,15 @@ def validation_step(self, batch, batch_idx=None) -> torch.Tensor:
 
         return self.forward_step(batch)
 
+    def get_inference_wrapper(self, inference_wrapper_config) -> torch.Tensor:
+        # get mcore, unwrapped model
+        mcore_model = self.module
+        if isinstance(mcore_model, DDP):
+            mcore_model = self.module.module
+
+        model_inference_wrapper = GPTInferenceWrapper(mcore_model, inference_wrapper_config)
```

Comment:
`inference_wrapper_config` could be provided automatically? We should be able to infer all fields

Reply:
What do mean by provided automatically? Can you explain more?

Reply:
Updated the code for `inference_wrapper_config` to be provided automatically.